llvm/tools/llvm-mca/Views/InstructionInfoView.cpp

```diff
@@ -141,6 +243,34 @@ void InstructionInfoView::collectData(
     IIVDEntry.mayLoad = MCDesc.mayLoad();
     IIVDEntry.mayStore = MCDesc.mayStore();
     IIVDEntry.hasUnmodeledSideEffects = MCDesc.hasUnmodeledSideEffects();
+
+    if (PrintFullInfo) {
+      // Get latency with bypass
+      IIVDEntry.Bypass =
+          IIVDEntry.Latency - MCSchedModel::getBypassDelayCycles(STI, SCDesc);
+      IIVDEntry.OpcodeName = (std::string)MCII.getName(Inst.getOpcode());
```

Comment:
please avoid casting to `std::string`, after you change `OpcodeName` to StringRef.